source/by-role/developer/common/adding-a-new-language.rst

```diff
@@ -0,0 +1,217 @@
+:is-up-to-date: True
+:last-updated: 4.1.2
+
+.. index:: Adding a New Language, Language Support
+
+.. _adding-a-new-language:
+
+=====================
+Adding a New Language
+=====================
+Crafter Studio has been translated into a number of languages.  Here's a list of currently supported languages along with the language codes used in Studio:
+
+* ``en`` - English
+* ``sp`` - Spanish
+* ``de`` - German
+* ``kr`` - Korean
+
+To reach more users, additional languages may need to be added in Crafter Studio.
+
+**Here are the steps to add a new language in Crafter Studio:**
+
+#. Add the new language to the ``get-available-languages`` API
+#. Update templates to add the new language imports into the runtime (i.e. via script[src] elements
```

Comment:
```suggestion
#. Update templates to add the new language imports into the runtime (i.e. via script[src] elements)
```

It would make sense to have this together with the "Add translation files for legacy forms and content type..."